bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableDataClient.java

```diff
@@ -36,12 +36,9 @@
  */
 public class BigtableDataClient {
```

Comment:
I'm starting to get lost in the wrappers. Not necessarily in this PR, but we add some doc and the layering

Reply:
I have a PR in progress to remove this wrapper.